webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/comet/AsyncDeployment.scala

```diff
@@ -250,26 +266,73 @@ class AsyncDeployment extends CometActor with CometListener with Loggable {
     } else NodeSeq.Empty
   }
 
+  private def showProperties:  NodeSeq = {
+    def tooltipContent(error: String): String                                = {
+      s"<h4 class='tags-tooltip-title'>Properties error</h4><div class='tooltip-inner-content'><pre class=\"code\">${StringEscapeUtils
+          .escapeHtml4(error)}</pre></div>"
+    }
+    def nodeBtnId(node: NodeId):       String                                = {
+      s"status-node-${StringEscapeUtils.escapeHtml4(node.value)}"
+    }
+    // The "a" tag has special display in the content, we need a button that opens link in new tab
+    def nodeLinkScript(node: NodeId):  NodeSeq                               = {
+      val link  = linkUtil.nodeLink(node)
+      val btnId = nodeBtnId(node)
+      scriptLinkButton(btnId, link)
+    }
+    val allNodes = nodeFactRepo.getAll()(QueryContext.systemQC).runNow
+    val nodesErrors:                   Map[MinimalNodeFactInterface, String] = propertiesStatus.flatMap {
+      case (_, _: SuccessNodePropertyHierarchy)     => None
+      case (nodeId, f: FailedNodePropertyHierarchy) => allNodes.get(nodeId).map(_ -> f.getMessage)
+    }
+    nodesErrors.toList match {
+      case Nil    => NodeSeq.Empty
+      case errors =>
+        <li class="card border-start-0 border-end-0 border-top-0">
+          <div class="card-body">
+            <h5 class="card-title d-flex justify-content-between">
+              Node configuration errors
+              <span class="badge bg-danger">{errors.size}</span>
+            </h5>
+            <ul class="list-group">
+            {
+          NodeSeq.fromSeq(
+            errors.map {
+              case (n, err) =>
+                <li class="list-group-item d-flex justify-content-between align-items-center">
+                  <button id={nodeBtnId(n.id)} class="btn btn-link text-start">
+                    {StringEscapeUtils.escapeHtml4(n.fqdn)} <i class="fa fa-external-link"/>
+                  </button>
```

Comment:
no we would need additional logic in both frontend (here to add the a url hash, and in the Elm app for groups), it would not be trivial, we could do it later in another PR